query_test.go

```diff
@@ -164,3 +164,17 @@ func TestSetExcludeScheme(t *testing.T) {
 		q.SetExcludeScheme(mode)
 	}
 }
+
+func TestAddTagExclude(t *testing.T) {
```

Comment:
It would be good to have a test that check that this actually does something, rather than just not throw an error.

Reply:
I can if you want, but this tests the functionality of notmuch itself and not actually this wrapper library.

Assuming we don't get an error, it's the responsibility of the notmuch test suite to validate that it does what it says on the tin.

But I will defer to whatever you want, so what do you want me to do?

Reply:
I guess I'm fine with leaving it as is.